app/src/main/java/com/example/habitsmasher/HabitEvent.java

```diff
@@ -78,7 +73,7 @@ public void setComment(String comment) {
      * Gets the picture URL of a habit event
      * @return pictureURL: The URL of the picture of the habit event
      */
-    @PropertyName("picture")
+    // @PropertyName("picture")
```

Comment:
Do we plan on adding this back later? Is this why it's sticking around?

Reply:
Yep!